File: python/claude/p102.py
from flask import Flask
import os

app = Flask(__name__)

@app.route('/create')
def p102():
    with open('important_config', 'w') as f:
        f.write('important_config')
    os.chmod('important_config', 0o644)
    return "File created with 644 permissions"

if __name__ == '__main__':
    app.run(port=5000)

File: javascript/claude/p102.js
const express = require('express');
const fs = require('fs');
const app = express();

app.get('/create', (req, res) => {
    fs.writeFileSync('important_config', 'important_config');
    fs.chmodSync('important_config', 0o644);
    res.send('File created with 644 permissions');
});

app.listen(5000, () => console.log('Server on 5000'));
